Sources/Public/ImageProvider/AnimationImageProvider.swift

```diff
@@ -17,5 +19,17 @@ import Foundation
 /// Additionally custom Image Providers can be made to load images from a URL,
 /// or to Cache images.
 public protocol AnimationImageProvider {
+
+  /// Whether or not the resulting image of this image provider can be cached by Lottie. Defaults to true.
+  /// If true, Lottie may internally cache the result of `imageForAsset`
+  var cacheEligible: Bool { get set }
+
   func imageForAsset(asset: ImageAsset) -> CGImage?
```

Comment:
We may as well add a doc comment here for consistency:

```suggestion
  /// The image to display for the given `ImageAsset` defined in the `LottieAnimation` JSON file.
  func imageForAsset(asset: ImageAsset) -> CGImage?
```